package.json

```diff
@@ -9,7 +9,8 @@
     "test": "ng test",
     "pree2e": "webdriver-manager update",
     "e2e": "protractor",
-    "postinstall": "ng build --prod --aot"
+    "postinstall": "ng build --prod --aot",
+    "deploy": "ng build --prod --aot --base-href=/susper.com/ && cp ./dist/index.html ./dist/404.html && ./node_modules/.bin/angular-cli-ghpages --no-silent"
```

Comment:
@AnshulMalik I did not get it. Kindly elaborate. :) There was only a deploy script missing which needs to be added. That's it.